homepage/homepage.go

```diff
@@ -82,7 +82,14 @@ func handle(w http.ResponseWriter, req *http.Request, rend RenderClient, zcli Ze
 	releaseCalResp, err := bcli.GetReleaseCalendar(ctx, userAccessToken, dateFromDay, dateFromMonth, dateFromYear)
 	releaseCalModelData := mapper.ReleaseCalendar(releaseCalResp)
 
-	m := mapper.Homepage(localeCode, mappedMainFigures, releaseCalModelData)
+	// Get homepage data from Zebedee
+	homepageContent, err := zcli.GetHomepageContent(ctx, userAccessToken, HomepagePath)
+	if err != nil {
+		log.Event(ctx, "error getting homepage data from client", log.Error(err), log.Data{"content-path": HomepagePath})
```

Comment:
I believe this as it stands currently means that the other homepage content like the Main Figures data can continue to be passed to the renderer and the page would still load - albeit without the In Focus section. It felt to me that still being able to render something, if not all of the homepage, would be preferable to not doing so. I don't think there have been any discussions on this specifically though, so it may not be a decision for me to make. If you think it's worthwhile still to error out of the function, or there's something I've perhaps not considered as a result of not returning here, let me know